kalite/updates/management/commands/videoscan.py

```diff
@@ -1,108 +0,0 @@
-"""
```

Comment:
We might want to have a 'import content' button on the download page? Or we can just require people to run setup.